### ISSUE TITLE
Maybe add v_ before variables

### ISSUE THREAD

**Issue by jkiviluo** [spine-tools/SpineOpt.jl#605]
Equations in code could be easier to read if variables would have v_ before them. TBD.

**Comment by clizbe** [comment]
@jkiviluo Have we made a decision on this?

**Comment by tarskul** [comment]
This is being discussed in issue #800

**Comment by tarskul** [comment]
Woops too soon, there is a difference between documentation and 
code.

**Comment by jkiviluo** [comment]
That wouldn't be needed if the IDE would differentiate (in color) between variables and parameters. I haven't really tried to set it up, but at least I don't see difference in PyCharm out of the box.

Another question is that how other new SpineOpt users feel about this - would it help to have v_ before variables (assuming there is no easy way to distinguish through IDE)?

**Comment by DillonJ** [comment]
I guess the problem here is that the distinction between variables and parameters is a JuMP thing and not a Julia language thing. In our constraint equations a variable is just an array. On top of that, there is the SpineInterface convenience functions that are sometimes parameters but sometimes indices. In the context of a constraint equation, one could differentiate thanks to the [()] convention for parameter value calls - but this isn't always used. 

Using some kind of `v_` convention probably would be helpful, but a headache to implement

**Comment by clizbe** [comment]
@nhniina Maybe we can include this in your structural changes?

**Comment by nhniina** [comment]
Yes, this would impact the names of the output entities in the input database.
